package.json

```diff
@@ -59,7 +59,7 @@
     "ses": "^0.11.0",
     "shx": "^0.3.2",
     "source-map-support": "^0.5.19",
-    "ts-proto": "^1.53.0",
+    "ts-proto": "=1.61.0",
```

Comment:
`=` is not actually a defined prefix for npm. See https://docs.npmjs.com/cli/v6/configuring-npm/package-json#dependencies

> `version` Must match `version` exactly

Reply:
Good to know, thanks. This should go back to `^` anyways when https://github.com/stephenh/ts-proto/issues/218 is resolved.

Reply:
> = Equal. If no operator is specified, then equality is assumed, so this operator is optional, but MAY be included.

https://docs.npmjs.com/cli/v6/using-npm/semver#ranges